examples/ipython/demo.md

```diff
@@ -45,29 +59,30 @@ the Jupyter notebook for local interaction as follows.
     nix-shell -I nixpkgs="https://github.com/NixOS/nixpkgs-channels/archive/cc6cf0a96a627e678ffc996a8f9d1416200d6c81.tar.gz" \
     -p "python3.withPackages(ps: [ps.notebook ps.numpy ps.matplotlib ps.pandas ps.pyarrow ps.networkx])"
     ```
-- With Homebrew
-    ```sh
-    brew install jupyter numpy matplotlib pandas apache-arrow networkx
-    ```
-- Via Virtual Env
+- With `venv`
     ```sh
-    virtualenv --system-site-packages venv
+    python3 -m venv venv
     source venv/bin/activate
-    python3 -m pip install -r requirements.txt
+    # Verify that the python from venv is used:
+    which python
+    python -m pip install -r requirements.txt
```

Comment:
I had to `pip install wheel` first, otherwise this would error out with
```
  ERROR: Command errored out with exit status 1:
   command: /usr/local/src/event-horizon/vast/examples/ipython/venv/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-c8gt70b_/pyrsistent/setup.py'"'"'; __file__='"'"'/tmp/pip-install-c8gt70b_/pyrsistent/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-23725vvk
       cwd: /tmp/pip-install-c8gt70b_/pyrsistent/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for pyrsistent
```

Reply:
FWIW, the installation worked fine on macOS 10.15.4.